### PR TITLE
Expand Gateway audit information.

### LINKED ISSUE #69: Add to Gateway audit description

### PR DESCRIPTION
**Pull request by no-reply**
Fixes #69.

### CODE REVIEW

File: otm-gateway.html [hunk 1]
```diff
@@ -223,15 +223,46 @@ <h3><dfn>Deposit Object</dfn></h3>
...
           <pre class="example" title="Object Audit JSON">
-            {
-            }
+            [
+              "object-id": "af48c3d",
+              "deposits": [
+                {
+                   "version" "20190702T201500.001",
+                   "gateway-errors": null,
+                   "status": "ACCEPTED",
```
Comment (bbranan) [inline otm-gateway.html]:
```suggestion
                   "status": "DEPOSIT_ACCEPTED",
```

File: otm-gateway.html [hunk 2]
```diff
@@ -223,15 +223,46 @@ <h3><dfn>Deposit Object</dfn></h3>
...
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/{object-id}/audit</code></li>
+          <li>Request Path: <code>/{object-id}/audit ? versionId=</code></li>
           <li>Response Body: <code>JSON</code></li>
+            <ul>
+              <li><code>object-id</code>: The id of the object this describes.</li>
+              <li><code>deposits</code>: Data about the deposited versions; this exposes any errors within the Gateway (e.g. failed <em>bag</em> validation) and the <a href="otm-bridge.html#get-deposit-status">Bridge status<a>.
+                <ul>
+                  <li><code>version</code>: The object version this deposit corresponds to.</li>
+                  <li><code>gateway-errors</code>: Any a human readable description of any errors encountered by the gateway.</li>
```
Comment (bbranan) [inline otm-gateway.html]:
```suggestion
                  <li><code>gateway-errors</code>: Human readable description of any errors encountered by the gateway.</li>
```

File: otm-gateway.html [hunk 3]
```diff
@@ -223,15 +223,46 @@ <h3><dfn>Deposit Object</dfn></h3>
 
       <section>
         <h3><dfn>Get Object Audit</dfn></h3>
-        <p></p>
+        <p>Describes the status and history of the object's preservation. For more information about auditing, see <a href="audit-appendix.html">OTM Appendix - Audit Events</a>.</p>
+        <p>The information exposed by this endpoint is intended to be informational, allowing repository administrators
+          to monitor content and resolve issues encountered during deposit. The Gateway MUST report any errors that would
```
Comment (bbranan) [inline otm-gateway.html]:
```suggestion
          to monitor deposit activity and resolve issues encountered during the deposit process. The Gateway MUST report any errors that would
```

File: otm-gateway.html [hunk 4]
```diff
@@ -223,15 +223,46 @@ <h3><dfn>Deposit Object</dfn></h3>
...
+          prevent it from requesting deposit via the Bridge in the <code>gateway-errors</code> field. The Gateway MAY
+          choose to poll status information from the Bridge asynchronously, and is not required to to provide real-time
+          information. The repository SHOULD NOT rely on the information in fields other than <code>gateway-errors</code>
+          to be up-to-date in real time.
+        </p>
         <ul>
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/{object-id}/audit</code></li>
+          <li>Request Path: <code>/{object-id}/audit ? versionId=</code></li>
```
Comment (bbranan) [inline otm-gateway.html:236]: Is `versionId` required or optional?

**Review by bbranan — CHANGES_REQUESTED** [review]
This is an interesting combination of deposit status information and audit log data. It's worth noting that the `Get Deposit Status` endpoint of the bridge only provides information about filegroups that are in the deposit process. After the deposit has completed, the calls would need to hit `Get Deposited Details`.  So the implementation of this method will likely need to touch at least 3 different bridge endpoints. That may be worth calling out.

File: otm-gateway.html (hunk 4 above)
Comment (no-reply) [reply otm-gateway.html:236]: optional, i think; where omitting it returns data for all versions

**Pull request merged**